ERROR Caught unhandled exception: Missing environment variable: CONDA_EXE

```log
Configuring PlaidML build environment

conda found at: C:\Users\cloud\Miniconda3\Library\bin\conda.BAT


Your build is configured.
Use the following to run all unit tests:

bazelisk test //...

INFO: Options provided by the client:
  Inherited 'common' options: --isatty=1 --terminal_columns=140
INFO: Options provided by the client:
  'build' options: --python_path=C:/Users/cloud/Miniconda3/python.exe
INFO: Reading rc options for 'build' from d:\machine_learning\plaidml\bzl\bazel.rc:
  'build' options: -c opt --spawn_strategy=standalone --genrule_strategy=standalone --announce_rc --nolegacy_external_runfiles --distinct_host_configuration=false --enable_platform_specific_config --action_env=LC_ALL --action_env=LANG --action_env=NO_PROXY --action_env=HTTP_PROXY --action_env=HTTPS_PROXY --action_env=no_proxy --action_env=http_proxy --action_env=https_proxy --define=version=0.0.0.dev0
INFO: Found applicable config definition build:windows in file d:\machine_learning\plaidml\bzl\bazel.rc: --build_tag_filters=-darwin,-docs,-latex,-linux,-metal,-pytorch --action_env=CONDA_EXE --action_env=NUMBER_OF_PROCESSORS --action_env=PLAIDML_SETTINGS --cxxopt=/std:c++17 --cxxopt=/Zc:__cplusplus
================================================================================
LIBXSMM release-1.12.1 (Windows_NT)
--------------------------------------------------------------------------------
Compiler: gcc, and g++
C / C++ target: <compiler default>
Fortran Compiler is disabled or missing: no Fortran interface is built!
--------------------------------------------------------------------------------
--- LIBXSMM build log
INFO: Analyzed target //plaidml:wheel (11 packages loaded, 18105 targets configured).
INFO: Found 1 target...
INFO: From Linking external/jsonnet/cmd/jsonnet.exe:
LINK : warning LNK4044: unrecognized option '/lm'; ignored
LINK : warning LNK4044: unrecognized option '/lm'; ignored
INFO: From Compiling Jsonnet to JSON for plaidml_link:
bazel-out/x64_windows-opt/bin/plaidml/plaidml.def
bazel-out/x64_windows-opt/bin/plaidml/plaidml.ld
bazel-out/x64_windows-opt/bin/plaidml/plaidml.lds
INFO: From Compiling external/boost/libs/regex/src/wc_regex_traits.cpp:
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xstring(2388): warning C4244: 'argument': conversion from '_Ty' to 'const _Elem', possible loss of data
        with
        [
            _Ty=const wchar_t
        ]
        and
        [
            _Elem=char
        ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xstring(2399): note: see reference to function template instantiation 'void std::basic_string<char,std::char_traits<char>,std::allocator<char>>::_Construct<_Ty*>(_Iter,const _Iter,std::input_iterator_tag)' being compiled
        with
        [
            _Ty=const wchar_t,
            _Iter=const wchar_t *
        ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xstring(2399): note: see reference to function template instantiation 'void std::basic_string<char,std::char_traits<char>,std::allocator<char>>::_Construct<_Ty*>(_Iter,const _Iter,std::input_iterator_tag)' being compiled
        with
        [
            _Ty=const wchar_t,
            _Iter=const wchar_t *
        ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xstring(2379): note: see reference to function template instantiation 'void std::basic_string<char,std::char_traits<char>,std::allocator<char>>::_Construct<_Ty*>(const _Iter,const _Iter,std::forward_iterator_tag)' being compiled
        with
        [
            _Ty=const wchar_t,
            _Iter=const wchar_t *
        ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xstring(2379): note: see reference to function template instantiation 'void std::basic_string<char,std::char_traits<char>,std::allocator<char>>::_Construct<_Ty*>(const _Iter,const _Iter,std::forward_iterator_tag)' being compiled
        with
        [
            _Ty=const wchar_t,
            _Iter=const wchar_t *
        ]
external/boost/libs/regex/src/wc_regex_traits.cpp(228): note: see reference to function template instantiation 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>::basic_string<const wchar_t*,void>(_Iter,_Iter,const _Alloc &)' being compiled
        with
        [
            _Iter=const wchar_t *,
            _Alloc=std::allocator<char>
        ]
external/boost/libs/regex/src/wc_regex_traits.cpp(228): note: see reference to function template instantiation 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>::basic_string<const wchar_t*,void>(_Iter,_Iter,const _Alloc &)' being compiled
        with
        [
            _Iter=const wchar_t *,
            _Alloc=std::allocator<char>
        ]
ERROR: D:/machine_learning/plaidml/tile/targets/cpu/BUILD:40:1: heatmap tile/targets/cpu/heatmap.cc failed (Exit 1)
Caught unhandled exception: Missing environment variable: CONDA_EXE
Target //plaidml:wheel failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 2671.115s, Critical Path: 19.50s
INFO: 381 processes: 381 local.
FAILED: Build did NOT complete successfully
```

By manually modifying https://github.com/plaidml/plaidml/blob/f2d57996958ebba7f147a0f7e00550928b6e661b/tools/conda_run/conda_windows.cc#L90 I can walkaround this issue. And the CONDA_EXE is correctly set in my case.

--- a/tools/heatmap/build_defs.bzl
+++ b/tools/heatmap/build_defs.bzl
@@ -11,6 +11,7 @@ def _heatmap_impl(ctx):
         inputs = [ctx.file.csv, ctx.file.template],
         outputs = [ctx.outputs.out],
         tools = [ctx.executable._tool],
+        use_default_shell_env = True,
     )
 
     return DefaultInfo(files = depset([ctx.outputs.out]))

